### ISSUE TITLE
Make the sign-up page create new user accounts

### ISSUE THREAD

**Issue by mgatland** [girlcodeakl/girlcodeapp#12]
Issues #9 and #10 need to be done before this

make a super basic sign up system

Act 1:
- [ ] Modify sign-up.html so when you click 'submit', it makes a POST request to a special URL "/users". You can save time by looking at post.html – your code will be mostly the same.
- [ ] Test - in your web browser, open the Developer Tools, then try clicking the 'submit' button on sign-up.html - you should see an error like 'cannot POST /signup'.

Act 2:
- [ ] Edit index.js to make the server respond to POST requests at /signup. You can copy and paste the code that responds to /post to get started, look for this:  `app.post('/ideas'`…

- [ ] Change to code so that instead of adding an idea to the ideas list, it adds a user to the user list. 
- [ ] You'll also need to create an empty list of users, so it has somewhere to put them! Create it similar to how we created the empty list of ideas, in index.js with some code like this: `var users = [];`

- [ ] when you add a new user to the list, save their name and password.

- [ ] Finally, make it so the server (index.js) responds to people trying to GET the address "/users" as well.
- [ ] find and copy the code from when someone tries to GET /posts.
- [ ] modify it to return the users instead of the posts
- [ ] test: in your web browser, go to http://localhost:300/users and you should now see a list of users (or an empty list like this `[]`)
- [ ] whoa, can you see all the passwords? Put a warning note on sign-up.html that tells people not to user their real password because it's publically viewable!

**Comment by mgatland** [comment]
Maybe we'll start by giving the server (index.js) a list of users, i.e.
`var users = [];`

When you sign up, we'll POST to a new url like /users. Then the server will save the new user in much the same way that we save a new idea.

- [ ] The server keeps a list of users
- [ ] The sign-up page can be used to POST a new user, and the server will save it to the list

**Comment by mgatland** [comment]
now out of scope
